package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-metrics",
-  "version": "0.16.0",
```

Comment:
Revert

Reply:
why, don't want to create a new release for that?

Reply:
This is up to the maintainers. Please remove.

Reply:
removing package-lock yes (and adding to gitignore next then) - although it is recommended to checkout package-lock in general

Reply:
removed package-lock and added to gitignore

Reply:
reverted package bump then